pisek/__main__.py

```diff
@@ -158,7 +158,7 @@ def add_argument_dataset(parser):
         "--inputs",
         "-n",
         type=int,
-        default=5,
+        default=1,
```

Comment:
Mostly as a preperation for #168. While repeating 5 times is much more safer, if you write in your generator ``repeat=2`` it would be quite surprising to generate it 10 times.

Reply:
Sure, but you'll unlikely to use `repeat` at all for Kasiopea or KSP-style tasks.

Reply:
Sure, but we intend to have this generator for CMS style tasks.

Reply:
But we still intend to use Pisek for Kasiopea, right?

Reply:
Sure but you can just add `-n5` if you want?

Reply:
I think 5 was a useful default.

Reply:
I also agree that 5 is a useful default for opendata tasks. But I think it's terrible default for CMS.

Reply:
Maybe we could load the default from the config file, with the expectation that it'll always be inherited from the task type config. Alternatively, we could change it based on `fail_mode`.